graphql/schema.graphql

```diff
@@ -1,7 +1,12 @@
 type Query {
-  suggestAddresses(searchTerm: String!): [AddressSuggestion!]!
+  suggestAddresses(
```

Comment:
since we are renaming things...... what do you think about `addressSuggestions`?

Reply:
Sure!!! Renaming the address query made it so much clear :satisfaction: